### ISSUE TITLE
2.5.0-beta3 release plan 

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1931]
- #1914
- #1947 
	- #1916 (d5ae6f64362e7ef5d734df98d81972e2767ddc26)
	- #1923 (5f381d909dd2be6eaeb7bf6107943453292063c6)
 	- #1921 (851c2737a9ce60ee86164fa1b8fba5b6135206d9)
    - #1912 (9a5369ba963247cfed3287454aafe7508b6b1189)
- #1533 
- #1934  
- #1910 
- #1906
- #1950 
- #1956
- #1952 
- #1951
- #1797

**Comment by joeparsons** [comment]
## Pick List

```
git cherry-pick 8e13050d8d09c1e461c92ba6b7429a342f6d0cd3
git cherry-pick d5ae6f64362e7ef5d734df98d81972e2767ddc26
git cherry-pick 5f381d909dd2be6eaeb7bf6107943453292063c6
git cherry-pick 851c2737a9ce60ee86164fa1b8fba5b6135206d9
git cherry-pick 9a5369ba963247cfed3287454aafe7508b6b1189
git cherry-pick 3fec6ff078d63d23794da96f4dccc7ef610db07a
git cherry-pick 5acc0998d59ca66c55dbd3ce18a3f5643e9c6f5c
git cherry-pick 0262474dcd89987640acd9e688b4c996cceb88f5
git cherry-pick 49196baaa29efee957aa9481801d2cca65b72f30
git cherry-pick fc51b067e99df24cbde7826475573f834bec3b0a
git cherry-pick b46cea602518531a15c1982ab199ee067c068a68
git cherry-pick e83485516a2e797ada626c18713e13d47a531dff
git cherry-pick 0203cc3b09c33e14aa9dcda7a44eeb5bbcac0927
git cherry-pick 8ebb8de5c11c117eb33fd75560c2c30860356cbf
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.5.0-beta2..HEAD --pretty='format:- %s'`

```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

- Closes #1763, Closes #1918 - Adding config for optionally importing archived paragraphs. (#1797)
- Update drupal/block_class requirement from 2.0.3 to 2.0.7 (#1951)
- Update drupal/metatag requirement from 1.21.0 to 1.22.0 (#1952)
- Update Arizona Bootstrap to 2.0.18 (#1956)
- Closes #1830: Update AZ Event Default view display to fix heading tags, reorder content (#1950)
- Update drupal/field_group requirement from 3.2.0 to 3.3.0 (#1906)
- Fixes #1877 Snapshot only Necessary Config on Override (#1910)
- Closes #1933 Remove migrate_plus.migration prefix on migration YAML filenames (#1934)
- Closes #1530 Adding theme-debug tooling for lando and ddev. (#1533)
- Closes #1911 preprocess embedded entities before passing to text_format_recognizer
- Closes #1920: Adding uaqs_hero_carousel to view mapping for views paragraph migration.
- Closes #1922 Add ability to migrate uaqs_alphabetical_listing view paragraphs
- Closes #1915 Adding default_value for view mode static map
- Update drupal/xmlsitemap requirement from 1.3.0 to 1.4.0 (#1914)
- Back to dev.
```

**Comment by joeparsons** [comment]
Released: https://github.com/az-digital/az_quickstart/releases/tag/2.5.0-beta3
